crates/providers/providers-alloy/Cargo.toml

```
@@ -58,3 +58,5 @@ metrics = [ "dep:metrics", "kona-derive/metrics" ]
 
 [dev-dependencies]
 tokio.workspace = true
+httpmock = "0.8.2"
```

When we're working with workspaces, it is usually better to pull dependency versions at the top-level `kona/Cargo.toml` (so that all crates use the same dependency version). Here, you can just use:
```suggestion
httpmock.workspace = true
```